config.json

```diff
@@ -1,4 +1,5 @@
 {
     "CAIRO_LANG": "0.10.3",
-    "STARKNET_DEVNET": "0.4.4"
+    "STARKNET_DEVNET": "0.4.4",
+    "STARKNET_DEVNET_CAIRO_VM" : "python"
```

Comment:
This could be defined in constants.ts. We only define versions/values here if they are to be used outside of the Typescript part of the plugin.

Reply:
changed